src/components/app-rating/AppRating.jsx

```diff
@@ -0,0 +1,56 @@
+import { useTranslation } from 'react-i18next'
+import StarSharp from '@mui/icons-material/StarSharp'
+import Box from '@mui/material/Box'
+import Rating from '@mui/material/Rating'
+import Typography from '@mui/material/Typography'
+
+import { styles } from '~/components/app-rating/AppRating.styles'
+
+const starsSize = { small: 'body2', medium: 'body1', large: 'h6' }
+
+const AppRating = ({ value = '', smallNumber, bigNumber, size = 'small', withBackground, mobile, reviews, spacing = '1px', ...props }) => {
+  const { t } = useTranslation()
+  const optionalStyles = {
+    backgroundColor: withBackground && 'primary.50',
+    flexDirection: bigNumber && 'column'
+  }
+
+  const bigNumberVariant = bigNumber && ( 
+    <Box data-testid='big-number-box' sx={ styles.bigNumber }>
+      { mobile && <StarSharp data-testid='star-icon' sx={ styles.starMobile } /> }
+      <Typography variant={ mobile ? 'h6' : 'h4' }  >
+        { value }
+      </Typography>
+    </Box>
+  ) 
+
+  return (
+    <Box sx={ [styles.root, optionalStyles] }>
+      { bigNumberVariant }
+      { !(bigNumber && mobile) && (
+        <Rating
+          emptyIcon={ <StarSharp fontSize='inherit' style={ styles.emptyIcon } /> }
+          name='feedback'
+          precision={ 1 }
+          sx={ { typography: starsSize[size],
+            '& .MuiRating-icon': {
+              mx: spacing
```

Comment:
As i see you dont pass spacing props anywhere, maybe better to remove it and move this sx to styles or we will need it in future?